ql/src/test/results/clientpositive/llap/external_jdbc_table_perf.q.out

```diff
@@ -6115,32 +6112,39 @@ WHERE "d_year" = 1999 AND "d_moy" BETWEEN 1 AND 3 AND "d_date_sk" IS NOT NULL) A
             Reduce Operator Tree:
               Merge Join Operator
                 condition map:
-                     Anti Join 0 to 1
+                     Left Outer Join 0 to 1
```

Comment:
Also not sure what caused this change in plan.

Reply:
This change is not actually related to this patch. Note that the test was disabled by default (HIVE-23910); it seems maybe a preliminary version of HIVE-23716 changed these q files and it should have not.

Reply:
Got it, make sense